elevation-plot/journey-specs.js

```diff
@@ -0,0 +1,75 @@
+import { getRoutePath } from './queries.js';
+
+/**
+ * Show route path specific information like elevation, consumption, speed etc.
+ * The temperature and maxspeed are not yet part of the API. This will be added soon.
+ * @param path {object} route path specification.
+ */
+export const displaySpecs = path => {
+  // The consumption, in kWh, of this route path segment.
+  document.getElementById('consumption').innerHTML = path.routePath.consumptionPerKm * 1000 + ' Wh/km';
+  // The elevation (altitude) in meters for this route path segment.
+  document.getElementById('height').innerHTML = path.routePath.elevation + ' m';
+  // The average speed, in km/h, for this route path segment.
+  document.getElementById('average-speed').innerHTML = path.routePath.avSpeed + ' km';
```

Comment:
Better to use one naming convention through the code - camelCase for IDs `averageSpeed` and a hyphen for class names `average-speed`. There are different conventions and one from Google says actually to use hyphen also for IDs, but we have it already as a camelCase. I think I personally also prefer it more, as then you can see the difference.